scripts/islandora_modules.sh

```diff
@@ -118,6 +89,7 @@ drush -y -u 1 en features context
 cd "$DRUPAL_HOME"/sites/all/modules || exit
 
 # Set variables for Islandora modules
+drush eval "variable_set('islandora_repository_pid', 'islandora:root')"
```

Comment:
@pc37utn this seems to have worked to some degree. I can now see the **Add an object to this Collection** link and it seems to function in relation to Fedora. However the frontend is just not showing the ingested object in that objects are not show in the grid and list view for the parent object.

I do the following:

In terminal
- `vagrant up`

In browser
- Browse to http://localhost:8000/collections/
- Login
- Browse to http://localhost:8000/collections/islandora/object/islandora%3Aroot
- Click **Manage**
- Click **Add an object to this Collection**
- Set **Collection PID** as _collections:alfa_
- Uncheck _Inherit collection policy?_
- Set **Children namespace** as _alfa_
- Check _islandora:sp_basic_image ~ Islandora Basic Image Content Model_ as **Allowable content models**
- Click **Next**
- Set **Children title** as _Alfa_
- Click **Ingest**

In terminal 
- `vagrant ssh`
- `cd /vhosts/digital/web/collections`
- `drush en islandora_scg`
- `drush iscgl --user=admin --quantity=5 --content_model=islandora:sp_basic_image --parent=collections:alfa --namespace=alfa`

Reply:
you have the namespace wrong.  it should be root:alfa

Reply:
and you may need to add the basic image content type to the islandora:root folder

Reply:
The issue was that the child objects ingested are not showing on http://localhost:8000/collections/islandora/object/collections%3Aalfa. I don't see why the namespace for collection record would effect that.

Given that, I've tried both suggestions and I'm seeing things as empty on both http://localhost:8000/collections/islandora/object/islandora%3Aroot and http://localhost:8000/collections/islandora/object/root%3Agarbage

Can you try to replicate and tell me what we are doing wrong? If there is a step I'm missing that I should do, please be specific and outline just as I do below.

----

In terminal
- `vagrant up`

In browser
- Browse to http://localhost:8000/collections/
- Login
- Browse to http://localhost:8000/collections/islandora/object/islandora%3Aroot
- Click **Manage**
- Click **Add an object to this Collection**
- Set **Collection PID** as _root:garbage_
- Uncheck _Inherit collection policy?_
- Set **Children namespace** as _garbage_
- Check _islandora:sp_basic_image ~ Islandora Basic Image Content Model_ as **Allowable content models**
- Click **Next**
- Set **Children title** as _Garbage_
- Click **Ingest**

In terminal 
- `vagrant ssh`
- `cd /vhosts/digital/web/collections`
- `drush en islandora_scg`
- `drush iscgl --user=admin --quantity=5 --content_model=islandora:sp_basic_image --parent=root:garbage --namespace=garbage`

Reply:
I found where the backend was set to solr and commented that out.  This might work.